### DISCUSSION TITLE
Uncaught (in promise) TypeError: e.response is undefined

### DISCUSSION THREAD

**Discussion by medoedoff** [DependencyTrack/dependency-track#958] [Q&A] [answered]
Greetings!

First deploy of dependency track by docker-compose according documentation https://docs.dependencytrack.org/getting-started/deploy-docker/

Trying to login as default admin user, but getting error:
```
Uncaught (in promise) TypeError: e.response is undefined
```

Could you help me, what's problem?

**Comment by niklasweimann** [comment]
@medoedoff Did you find a solution for this problem? 🤔

**Reply by medoedoff** [reply]
> @medoedoff Did you find a solution for this problem? 🤔

I am not sure, it was 3 years ago, I remember that I decided to not use dependency tracker cause it was quite buggy back in 2021

**Reply by niklasweimann** [reply, marked as answer]
Okay... After some testing it turned out that this error occurs when API_BASE_URL is not set properly.

For others who come to this issue:
I am using an nginx proxy which redirects all traffic from example.com/api/ to dtrack-server:8080/api/ and example.com/ to dtrack-frontend:8080 where dtrack-server and dtrack-frontend are docker containers within a network. For the frontend to work properly you need to set API_BASE_URL to https://example.com/. And you will need to wait until the dtrack-server is fully started otherwise you will also get the error described above.
